verifier/check_known_issues.py

```diff
@@ -287,6 +290,18 @@ def langname_brackets(test):
         return None
 
 
+def check_plural_rules_issues(test):
+    try:
+        input_data = test['input_data']
+        sample_string = input_data['sample']
+        # Plural rules for floating point values may not be supported
+        if sample_string.find('.') >= 0:
+            return knownIssueType.plural_rules_floating_point_sample
```

Comment:
You should also check that the final digit is `0`

Reply:
Probably check if all digits after "." are "0", not just the last one.

Reply:
Also changed so  test reported as "unsupported" by the executor are not reassigned as "known issues".

Reply:
Done